src/test/index.ts

```diff
@@ -98,3 +101,29 @@ describe('CreateUser Mutation', async () => {
     expect(response.data.errors[0].extensions.exception.code).to.be.equal(400);
   });
 });
+
+const loginInput = {
+  email: 'userteste2@email.com',
+  password: '1234abc'
+};
+describe('Login Mutation', async () => {
+  beforeEach(async () => {
+    await AppDataSource.getRepository(User);
+  });
+
+  it('should login', async () => {
+    const findUser = await AppDataSource.manager.findOneBy(User, {
+      email: loginInput.email
+    });
+    const response = await QueryLogin(loginInput);
+    const { email, name, birthDate } = response.data.data.login.user;
+    const token = response.data.data.login.token;
+    const decoded = jwt.verify(token, 'thisisasecretkey');
+    const tokenPayload = decoded as jwt.JwtPayload;
+    expect(email).to.be.equal(findUser.email);
+    expect(name).to.be.equal(findUser.name);
+    expect(birthDate).to.be.equal(findUser.birthDate);
+    expect(tokenPayload.email).to.be.equal(findUser.email);
+    expect(tokenPayload.userId).to.be.equal(findUser.id);
```

Comment:
Acho que o objeto `user` da resposta vc poderia checar com `to.be.deep.eq`, pra ser mais assertivo.